### PR TITLE
Quick Actions: FTUE Integration

### LINKED ISSUE #7684: Quick Action Menu + FTUE
There's a few menu actions that should interact with the help center, they are listed below and are only to fire if those tips have yet to be read by the user. 

1) background image - replace media - trigger the background image tip 
2) foreground image - replace image - trigger replace image tip 

## Acceptance criteria 
- the above functionality
 - some tests to validate it

### PR DESCRIPTION
**Pull request by maxyinger**
## Context
Part of Quick Actions work.

## Summary
This PR triggers relevant tips for fg and bg media when using the media replace quick action

## Relevant Technical Choices
NA

## User-facing changes
NA
-------------------------------
(This is all behind the quick actions experiment)
Triggers tips when FG Media Swap or BG Media Swap are clicked

https://user-images.githubusercontent.com/35983235/120345371-09de3900-c2c0-11eb-9cbd-62ba383573ae.mp4

https://user-images.githubusercontent.com/35983235/120345373-0a76cf80-c2c0-11eb-8776-60b75970a3e7.mp4

## Testing Instructions
enter this into your terminal to clear your read quick tips:
```
await wp.apiFetch({ path: '/web-stories/v1/users/me', method: 'POST', data: { meta: { web_stories_onboarding: {} }}})
```

Then add a bg image or fg image and click the replace media quick action. Should trigger a relevant ftue tip

### QA

- [x] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

### UAT

- [x] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #7684

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,6 +65,7 @@
     "big.js": "^6.1.1",
     "classnames": "^2.3.1",
     "colorthief": "^2.3.2",
+    "contains-path": "^1.0.0",
```
Comment (maxyinger) [inline package.json]: Didn't add this for anything relevant to this PR, but continually got an internal package dep error that it couldn't find this so just added it as a dep to appease that. Happy to remove this and the updates to the package-lock if no one else was running into this issue and it was just something wrong with my local.

Comment (swissspidy) [reply package.json]: That sounds odd 🤔  Haven't experienced this one so far.

Does this also happen after removing `node_modules` and doing an `npm install` again?

I'd really avoid adding random deps where possible.

Comment (maxyinger) [reply package.json]: sounds good. I'll revert both of these. I think it was just an issue with my local

File: assets/src/edit-story/components/helpCenter/index.js [hunk 2]
```diff
@@ -54,6 +55,20 @@ export const HelpCenter = () => {
...
+      actions.openToUnreadTip(KEYS.ADD_BACKGROUND_MEDIA);
+    }, [actions])
+  );
+
+  useStoryTriggerListener(
+    STORY_EVENTS.onReplaceForegroundMedia,
+    useCallback(() => {
+      actions.openToUnreadTip(KEYS.CROP_SELECTED_ELEMENTS);
+    }, [actions])
+  );
```
Comment (samwhale) [inline assets/src/edit-story/components/helpCenter/index.js:70]: This is so dope

**Review by samwhale — APPROVED** [review]
This was fun to play with.

One question unrelated to this pr: Should the Help Center close when we click away from it? 

thing

**Comment by maxyinger** [comment]
> One question unrelated to this pr: Should the Help Center close when we click away from it?

Nope. This was an active decision by design to not make it close so that users can reference the tip while they're performing the action they're learning from it

**Review by BrittanyIRL — APPROVED** [review]
Works well! 

There's 2 karma tests failing, those need to get fixed before merging. Otherwise, lgtm.

**Comment by samwhale** [comment]
@googlebot I consent.

File: assets/src/edit-story/components/canvas/karma/quickActions.karma.js [hunk 3]
```diff
@@ -44,14 +43,6 @@ describe('Quick Actions integration', () => {
     fixture.restore();
   });
 
-  describe('quick action menu should have no aXe accessibility violations', () => {
```
Comment (BrittanyIRL) [inline assets/src/edit-story/components/canvas/karma/quickActions.karma.js]: i think a rebase maybe went south here, these tests shouldn't get removed.

File: assets/src/edit-story/components/canvas/karma/quickActions.karma.js [hunk 4]
```diff
@@ -526,143 +517,6 @@ describe('Quick Actions integration', () => {
     });
   });
 
-  describe('text element selected', () => {
```
Comment (BrittanyIRL) [inline assets/src/edit-story/components/canvas/karma/quickActions.karma.js]: samesies here, shouldn't get removed

**Review by BrittanyIRL — CHANGES_REQUESTED** [review]
i'm just blocking for now cuz of a bad rebase 🙃

**Review by BrittanyIRL — APPROVED** [review]

**Pull request merged**